Frameworks/CoreFoundation/Base.subproj/CFInternal.h

```diff
@@ -707,11 +707,14 @@ CF_INLINE uintptr_t __CFISAForTypeID(CFTypeID typeID) {
 CF_INLINE bool __CF_IsBridgedObject(CFTypeRef obj) {
     CFRuntimeBase* object = (CFRuntimeBase*)obj;
     if (!object || 
-        (object->_cfisa == 0) || 
-        (object->_cfisa == (uintptr_t)(&_OBJC_CLASS__NSCFString)) || 
+        (object->_cfisa == 0)) {

```

Comment:
Remember that we are manually looking into the object's isa field here. Nil is comparison made on the pointer itself. A 0 isa means its a cf object that has not been bridged yet. 

---
In reply to: [69772935](https://github.com/Microsoft/WinObjC/pull/609#discussion_r69772935) [](ancestors = 69772935)